common/src/main/java/dev/architectury/event/events/common/LifecycleEvent.java

```diff
@@ -88,6 +89,14 @@
      * @see ServerLevelState#act(Level)
      */
     Event<ServerLevelState> SERVER_LEVEL_SAVE = EventFactory.createLoop();
+    /**
+     * Invoked once common setup has begun.
+     * <p> This happens during {@code FMLCommonSetupEvent} on Forge,
+     * or when Architectury API's entrypoint initialises on Fabric.
```

Comment:
```suggestion
     * or when Architectury API's client/server entrypoint initialises on Fabric.
```

Reply:
I don't think this is important and I think this may be even more confusing, fabric simply invokes the client and server entrypoints after the main one, so the main reason why we put our hook there is to be invoked after all of the main ones.

Reply:
Yes, but doesn't *this* imply that it's when Architectury API's `ModInitializer` is invoked?

Reply:
I mean this doesn’t say main entrypoint

Reply:
I feel like people might assume "entrypoint" would just mean "main" entrypoint, so Basique does have a point here, though I'm not sure whether "client/server entrypoint" is better terminology here

Reply:
I am accepting Basique's suggestion here, since I can't think of a better javadoc.